server/etcdserver/healthz_test.go

```diff
@@ -0,0 +1,326 @@
+// Copyright 2022 The etcd Authors
```

Comment:
Need to change to the right calendar year 2023. 

Could you run `make fix` and `make verify` and install any lint tools if it is missing in your dev box?

Reply:
`make fix` does not detect the 2023 errors. 
`make verify` is broken due to the protogen error `sed: 1: "s#import \(#import \("g ...": RE error: parentheses not balanced`, which also broken for the main branch. I don't know where the error is.

Reply:
> make fix does not detect the 2023 errors.

Feel free to create a issue. 

> make verify is broken due to the protogen error sed: 1: "s#import \(#import \("g ...": RE error: parentheses not balanced, which also broken for the main branch. I don't know where the error is.

The etcd scripts are not developed and tested in Mac so the `sed` version installed on Mac may not be the same. This should be reflected in the new contributor play book. 

Feel free to resolve the conversation if the comment is resolved.